packages/blade/src/components/Dropdown/types.ts

```diff
@@ -1,8 +1,40 @@
+import type React from 'react';
 import type { StyledPropsBlade } from '~components/Box/styledProps';
 import type { TestID } from '~utils/types';
 
 type DropdownProps = {
+  /**
+   * Control open / close state of the Dropdown component
+   *
+   * Make sure to not misuse this prop to create custom triggers. Use DropdownButton instead for such usecases.
```

Comment:
probably remove this line `Make sure to not misuse this prop to create custom triggers`? because somethings are better being ambiguous than explicit :D

Reply:
haha yes someone will read this and think: "Woahhh we can do that?? 😆"